yt_dlp/options.py

```diff
@@ -215,6 +224,22 @@ def _match_long_opt(self, opt):
                 return e.possibilities[0]
             raise
 
+    def format_option_help(self, formatter=None):
+        assert formatter, 'Formatter can not be None'
+        formatted_help = super().format_option_help(formatter=formatter)
+        formatter.indent()
+        heading = formatter.format_heading('Preset aliases')
```

Comment:
to be consistent with other headings
```suggestion
        heading = formatter.format_heading('Preset Aliases')
```